tools/generate_config_yaml_for_sharded_dst.py

```diff
@@ -0,0 +1,86 @@
+import pathlib
```

Comment:
I'd add a disclaimer docstring saying that this is only for older datasets, and not really intended for anything else